src/main/java/com/alvarium/annotators/sca/OsvBatchRequest.java

```diff
@@ -0,0 +1,41 @@
+package com.alvarium.annotators.sca;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.google.gson.annotations.SerializedName;
+
+public class OsvBatchRequest {
+    public List<Query> queries;
+
+    OsvBatchRequest(Map<String,String> packages) {
+        queries = new ArrayList<Query>();
+        for (Map.Entry<String, String> entry : packages.entrySet()) {
+            Package packageObj = new Package(entry.getKey());
+            queries.add(new Query(entry.getValue(), packageObj));
+        }
+    }
+
+    static class Query {
+        @SerializedName("package") // package is a reserved keyword in Java
+        public Package packageObj;

```

Comment:
According to the twitter java code guidelines that we follow for this project (https://github.com/twitter-archive/commons/blob/master/src/java/com/twitter/common/styleguide.md#dont-embed-metadata-in-variable-names): 

> Avoid embedding the field type in the field name.

> ```java
> // Bad.
> Map<Integer, User> idToUserMap;
> String valueString;
>
> // Good.
> Map<Integer, User> usersById;
> String value;
> ```

I propose maybe `packageInfo` or something similar ?